OA-PASS/pass-docker
# Make activemq destination configurable in Fedora

## Overview
* Makes it possible to configure Fedora to send to a queue or a topic, via `FCREPO_JMS_PUBLISHER`.
* Makes it possible to set the queue/topic name where Fedora sends messages, via `FCREPO_JMS_DESTINATION`
* Configures ActiveMQ to allow virtual topics
* Configures fedora, indexer, and deposit to use virtual topics/queues

## To test
* Do a `docker-compose pull`, `docker-compose up`
* Go to http://localhost:8161 and log into activemq (admin/admin).
* Go to the queues tab, and verify that there are three queues for virtual topic consumers
* Log in as a user (e.g. `nih-user`), and complete a new submission.
* Go to http://localhost:8161 and look at the statistics.  Were messages sent and consumed from the queues?
